observer/observer.go

```diff
@@ -106,3 +108,10 @@ func inferUtxoValue(tx *blockatlas.Tx, address string, coinIndex uint) {
 		}
 	}
 }
+
+func GetInterval(value int, minInterval, maxInterval time.Duration) time.Duration {
```

Comment:
`value int` - what does it mean in this context?
`GetInterval` - is this generic internal of block internal ? 

I'm not able to easily follow what does the function does.